pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -1104,8 +1239,9 @@ func Run(ps *pubsub.PubSub) {
 
 		pubEveNodeCert, err := ps.NewPublication(
 			pubsub.PublicationOptions{
-				AgentName: agentName,
-				TopicType: types.EveNodeCert{},
+				AgentName:  agentName,
+				TopicType:  types.EveNodeCert{},
+				Persistent: true,
```

Comment:
Why do you need this to be persistent? I shouldn't hurt but I wonder if there is some subtle ordering issue which requires it.

Reply:
removed

Reply:
Did you check that WiFi decryption doesn't depend on it first? I honestly don't know exactly what it depends on to decrypt the WiFi credentials.